app/views/users/nbp_wallet/connect.html.slim

```diff
@@ -0,0 +1,24 @@
+.header.mb-5
+  .std-heading
+    i.fa-solid.fa-wallet
+    =< t('.header')
+
+.container
+  .row
+    .col-8.col-md-4.col-lg-3
+      img.nbp_wallet_qr_code[src="#{nbp_wallet_qr_code_users_path}?truncated_reference=#{@truncated_reference}"]
+      br
+      br
+      = button_tag class: 'btn btn-primary regenerate-qr-code-button w-100' do
+        = t('.regenerate_code')
+    .col-8.col-md-6.m-3
```

Comment:
To avoid the spacing in the mobile view (and rather align it with the privacy note):

```suggestion
    .col-8.col-md-6.mt-3.ms-md-3.mt-md-0
```